src/components/refactor/dialogs/commons/modificationDialog.js

```diff
@@ -83,13 +96,18 @@ const ModificationDialog = ({
             aria-labelledby={titleId}
             {...dialogProps}
         >
-            <DialogTitle>
+            <DialogTitle className={headerPane ? classes.header : ''}>
```

Comment:
Maybe use clsx conditionnal rendering ?

Reply:
like this ?
className={clsx({
                    [classes.header]: headerPane !== undefined,
                })}